radar/platforms-and-services/azure-managed-identity.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Azure Managed Identity"
+ring: adopt
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, azure, security]
+---
+
+[Azure Managed Identities](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview) simplify identity management for applications and services running in the cloud. They provide a secure and seamless way for applications to access Azure resources without the need for explicit credentials. Managed Identities are tightly integrated with [Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/whatis) and eliminate the need for developers to manage and rotate authentication keys manually. This helps enhance security by reducing the risk associated with managing and storing sensitive credentials. Azure Managed Identities come in two types: System-assigned, which is tied to an Azure resource, and User-assigned, which can be associated with multiple resources.
```

Comment:
if we don't plan to put Entra ID in our radar I'd remove all references to it.

Reply:
We definitely need to add entra id soon

Reply:
I suggest to add it in 'assess'.

Reply:
Why? We use it actively (user management, service connection, etc.)